src/bp-forums/groups.php

```diff
@@ -268,7 +268,7 @@ public function edit_screen( $group = false ) {
 			// Get the first forum ID
 			if ( ! empty( $forum_ids ) ) {
 				$forum_id = (int) is_array( $forum_ids ) ? $forum_ids[0] : $forum_ids;
-			}
+			}	
```

Comment:
@kent-carlo These changes in not required. Can you verify changes before the final commit?